### PR TITLE
Move GITHUB_BASE definition to buildlib

### PR DESCRIPTION
**Pull request by locriandev**
Test run with custom job: https://saml.buildvm.hosts.prod.psi.bos.redhat.com:8888/job/hack/job/dpaolell/job/custom/56/console

### CODE REVIEW

File: pipeline-scripts/buildlib.groovy [hunk 1]
```diff
@@ -23,6 +24,7 @@ def initialize(test=false, regAws=false) {
 
     GITHUB_URLS = [:]
     GITHUB_BASE_PATHS = [:]
+    GITHUB_BASE = "git@github.com:openshift"
```
Comment (vfreex) [inline pipeline-scripts/buildlib.groovy:27]: Why is this needed? I saw it was already initialized at declaration.

**Review by vfreex — COMMENTED** [review]
One nit, otherwise LGTM

File: pipeline-scripts/buildlib.groovy (hunk 1 above)
Comment (locriandev) [reply pipeline-scripts/buildlib.groovy:27]: yeah thought that too, then I saw `GITHUB_URLS` and `GITHUB_BASE_PATHS` initialized twice as well, and my scarce Groovy knowledge made me worry. Shall I remove all those duplicated initializations?

**Comment by vfreex** [comment]
Thanks. Let's try it.
